sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExportersEndToEndTest.java

```diff
@@ -194,42 +201,44 @@ private static void generateLog(OpenTelemetry openTelemetry) {
             .emit();
     }
 
-    private static void validateSpan(TelemetryItem telemetryItem) {
+    private static void validateSpan(TelemetryItem telemetryItem) throws IOException {
```

Comment:
can remove now I think
```suggestion
    private static void validateSpan(TelemetryItem telemetryItem) {
```